### ISSUE TITLE
Centered modals and dual monitors

### ISSUE THREAD

**Issue by millette** [beakerbrowser/beaker#643]
Operation System: Debian Jessie
Beaker Version: any

I have two monitors and when I open a modal (dat picker, create dat, etc.) it opens centered accross the 2 screens instead of being centered on the monitor where Beaker lives.

I'm using the i3 tiling window manager.

This might help:
https://github.com/electron/electron/blob/master/docs/api/screen.md

**Comment by cdaringe** [comment]
i'll try this on my 2nd screen tomorrow.

sure enough, pfrazee has [already written code to handle exactly this](https://github.com/beakerbrowser/beaker/commit/8410b3d6aade5651f4e038a7f89e1e4c5db5a07b#diff-123d7d4687874aac4b56ce08f94bf61cR25), ~1 yr ago.

i just wrote some basic assertions against it--they seem to pass?  something more must be at play.  hard to test w/out that 2nd monitor :$

```js
var assert = require('assert')

function windowWithinBounds (windowState, bounds) {
  return windowState.x >= bounds.x &&
    windowState.y >= bounds.y &&
    windowState.x + windowState.width <= bounds.x + bounds.width &&
    windowState.y + windowState.height <= bounds.y + bounds.height
}

assert(windowWithinBounds({ x: 0, width: 100, y: 0, height: 100 }, { x: 0, y: 0, width: 100, height: 100 }))
assert(windowWithinBounds({ x: -1, width: 100, y: 0, height: 100 }, { x: 0, y: 0, width: 100, height: 100 }) == false)
assert(windowWithinBounds({ x: 0, y: 0, width: 101, height: 100 }, { x: 0, y: 0, width: 100, height: 100 }) == false)
assert(windowWithinBounds({ x: 0, y: 0, width: 100, height: 101 }, { x: 0, y: 0, width: 100, height: 100 }) == false)
```

**Comment by pfrazee** [comment]
Solved in 0.8
